cmd/substreams/pack.go

```diff
@@ -20,12 +24,33 @@ var packCmd = &cobra.Command{
 
 func init() {
 	rootCmd.AddCommand(packCmd)
+	packCmd.Flags().StringP("output-dir", "o", "src/pb", cli.FlagDescription(`
+		Optional flag to specify output directory to output generated .spkg file. If the received <package> argument is a local Substreams manifest file
+		(e.g. a local file ending with .yaml), the output folder will be made relative to it
+	`))
 }
 
 func runPack(cmd *cobra.Command, args []string) error {
+	outputDir := maybeGetString(cmd, "output-dir")
+
+	validOutputDirectorySpecified := true
+	fileInfo, err := os.Stat(outputDir)
+	if err != nil || !fileInfo.IsDir() {
```

Comment:
We should instead create the hierarchy of folders automatically with `os.MkdirAll` and not check if it exists and any I/O error will be caught right now in `os.MkdirAll` or later when writing the file.

Reply:
👍